frontends/concrete-python/concrete/fhe/mlir/converter.py

```diff
@@ -205,7 +235,14 @@ def process(self, graph: Graph):
         )
 
         for processor in pipeline:
-            processor.apply(graph)
+            if isinstance(processor, MultiGraphProcessor):
+                processor.apply_many(graphs)
+            elif isinstance(processor, GraphProcessor):
+                for graph in graphs.values():
+                    processor.apply(graph)
+            else:
+                error = "Unknown processor type."
+                raise RuntimeError(error)
```

Comment:
Maybe an assert instead of runtime error, this is an internal detail it should be asserted not reflected to users. Also, this is hard to get 100% coverage :grin: